ci/builders/README.md

```diff
@@ -1,69 +1,55 @@
+**Created:** 01/2023   /  **Last updated:** 04/2023
+
+## Glossary
+
+* **[recipes](https://github.com/luci/recipes-py)** - domain specific
+language for specifying sequences of subprocess calls in a cross-platform and
+testable way.
+* **Generator** - scripts in Dart, python or bash that combine the output of
+sub-builds to generate artifacts.
+* **Builder** - a combination of configuration, recipes and a given commit to
+build and test artifacts.
+* **Build** - a builder running with specific properties, repository and
+commit.
+* **[GN](https://gn.googlesource.com/gn/)** - a meta-build system that
+generates build files for [Ninja](https://ninja-build.org/).
+* **[Ninja](https://ninja-build.org)** - Ninja is a small build system with a
+focus on speed.
+* **CAS** - a service that stores arbitrary binary blobs addressed by (hash of)
+their content. It is specialized for low latency, high volume query/read/write
+operations.
 
 ## USAGE EXAMPLES
 
-Engine builds will be translated one to one using the build definition language with one file per build. The build definition files will be stored in the [flutter/engine/ci/builders](https://cs.opensource.google/flutter/engine/+/main:ci/builders/) directory.
+Engine build definition files using the Build Definition Language can be found in the
+[flutter/engine/ci/builders](https://cs.opensource.google/flutter/engine/+/main:ci/builders/) directory.
 
-The [engine orchestrator recipe](https://cs.opensource.google/flutter/recipes/+/main:recipes/engine_v2/) will read the file, shard builds, collect artifacts and upload them to the Google Cloud Storage bucket where they are downloaded by the Flutter Tool.
+The [engine orchestrator recipe](https://cs.opensource.google/flutter/recipes/+/main:recipes/engine_v2/)
+reads each file in that directory, shards their builds, collects artifacts and
+uploads them to the Google Cloud Storage bucket.
 
-Ci\_yaml file is the glue sticking all the components together. A new build will be added to the [.ci.yaml](https://cs.opensource.google/flutter/engine/+/main:.ci.yaml) file with a property pointing to the build definition file to be used by engine\_v2 recipes. The following is an example of a build configuration referencing [android\_aot\_engine.json](https://cs.opensource.google/flutter/engine/+/main:ci/builders/mac_android_aot_engine.json):
+The [.ci.yaml file](https://github.com/flutter/engine/blob/main/.ci.yaml) at the
+root of the `flutter/engine` repository puts all the components together.
+Builds are specified in that file using a property pointing to the build definition
+file to be used by engine\_v2 recipes. Full documentation of the `.ci.yaml` file format
+can be found [in the Cocoon repository here](https://github.com/flutter/cocoon/blob/main/CI_YAML.md).
 
-```
+The following is a sample build configuration referencing
+[android\_aot\_engine.json](https://github.com/flutter/engine/blob/main/ci/builders/mac_android_aot_engine.json)
```

Comment:
nit: if we want to be explicitly clear we should note that you need to omit the .json extension here as the recipe knows that this is expected to be json.